src/test/kotlin/ar/edu/unahur/obj2/vendedores/CentroDistribucionTest.kt

```diff
@@ -0,0 +1,120 @@
+package ar.edu.unahur.obj2.vendedores
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.DescribeSpec
+import io.kotest.matchers.collections.shouldBeEmpty
+import io.kotest.matchers.collections.shouldContain
+import io.kotest.matchers.shouldBe
+
+class CentroDistribucionTest : DescribeSpec({
+    /*Primero una clase grande que inheritea describespec*/
+    //--------Provincias-----------------/
+    val yapeyu = Provincia(99500)
+    val kayaki = Provincia(25500)
+    val babilonia = Provincia(250000)
+    val zorrillo = Provincia(50000)
+    //--------Ciudades-------------------/
+    val malabia = Ciudad(yapeyu)
+    val esperanto = Ciudad(babilonia)
+    val cachaquito = Ciudad(kayaki)
+    val frandela = Ciudad(zorrillo)
+    //--------Vendedores-----------------/
+    val provinciasEstefania = listOf(yapeyu,kayaki,zorrillo) //auxiliar
+    val ciudadesEduardo = listOf(malabia,esperanto,cachaquito,frandela) //auxiliar
+    val candelaria = VendedorFijo(frandela)
+    val estefania = Viajante(provinciasEstefania)
+    val eduardo = ComercioCorresponsal(ciudadesEduardo)
+    //-------Certificaciones---/
+    val certT20 = Certificacion(true,20)
+    val certT50 = Certificacion(true,50)
+    val certT100 = Certificacion(true,100)
+    val certF20 = Certificacion(false,20)
+    val certF50 = Certificacion(false,50)
+    val certF100 = Certificacion(false,100)
+    //--------Centro-Distribucion--------/
+    val vendedoresCD = mutableListOf(candelaria,estefania,eduardo)
+    val centroDistribuciones = CentroDistribucion(esperanto,vendedoresCD)
+    /*Ahora que estan todos los objetos creados, vienen los describe chiquitos*/
+
+
+
+    describe("Agregar vendedor."){
+        /*Ahora del ahora: vienen los IT chiquitos.*/
+        val vendedorNuevo = VendedorFijo(cachaquito)
+        it("Agregar un vendedor."){
+            centroDistribuciones.agregarVendedor(vendedorNuevo)
+
+        }
+        it("Agregar al mismo vendedor."){
+            shouldThrow<Exception>{ centroDistribuciones.agregarVendedor(vendedorNuevo) }
```

Comment:
Este test anda medio de casualidad: si lo corrés solo, no anda, pero si corrés el describe "Agregar vendedor" que incluye estos dos `it`, ahí sí. Eso es porque depende de que se ejecute el `.agregarVendedor(...)` del test que está justo arriba.

Está bueno que los tests no dependan entre ellos porque se vuelve complicado de mantener. En las próximas clases vamos a ver más sobre esto!

Reply:
Ya lo corregi. Pense que era buena implementacion hacer los valores y variables que se repiten entre "it" adentro de cada describe, ahora cada it tiene sus propias asignaciones.